packages/aragon-wrapper/src/rpc/handlers/addressIdentity.js

```diff
@@ -0,0 +1,14 @@
+export default function (request, proxy, wrapper) {
+  const [operation, address, providerName] = request
+  if (operation === 'resolve') {
+    return wrapper.resolveAddressIdentity(address, providerName)
+  }
+
+  if (operation === 'modify') {
+    return wrapper.requestAddressIdentityModification(address, providerName)
```

Comment:
This doesn't implement what the interface expects (should only emit if the actual modification was successful); we could do something like the [`transactions` observable and provide `accept()` and `reject()` methods](https://github.com/aragon/aragon.js/blob/master/packages/aragon-wrapper/src/index.js#L686) in [`requestAddressIdentityModification()`](https://github.com/aragon/aragon.js/pull/253/files#diff-5b914b5b526a57f89974fa2fe009ee80R521).